Remmber#6.4.py

```diff
@@ -0,0 +1,17 @@
+import imageio as iio
+
+# read an image
+im = iio.v3.imread('resources/code.png')
+height_weight = im.shape
+list_of_numbers = []
+for j in range(0, height_weight[1]):

```

Comment:
Replace these two for loops with a list comprehension with two iterators - 
https://stackoverflow.com/questions/18551458/how-to-frame-two-for-loops-in-list-comprehension-python